src/test/java/org/openrewrite/quarkus/Quarkus1to113MigrationTest.java

```diff
@@ -108,6 +110,7 @@ public void defaults(RecipeSpec spec) {
                 .activateRecipes("org.openrewrite.quarkus.Quarkus1to1_13Migration"));
         }
 
+        @DocumentExample
```

Comment:
@kunli2 Is there a second example here because of the `@Nested` test class? Is that expected / desired?